tests in TestCopyRange fail on ppc64le with "inappropriate ioctl for device"

From an attempted 1.8.0 build on fedora rawhide (with go 1.15, but using `-tags noquic` where necessary):

```
go test -buildmode pie -compiler gc -ldflags ' -extldflags '\''-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '\''' github.com/syncthing/syncthing/lib/fs
--- FAIL: TestCopyRange (1.28s)
    --- FAIL: TestCopyRange/tmp (1.28s)
        --- FAIL: TestCopyRange/tmp/ioctl (0.22s)
            --- FAIL: TestCopyRange/tmp/ioctl/append_to_end (0.02s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
            --- FAIL: TestCopyRange/tmp/ioctl/append_to_end_offsets_at_start (0.02s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
            --- FAIL: TestCopyRange/tmp/ioctl/overwrite_part_of_destination_region (0.02s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
            --- FAIL: TestCopyRange/tmp/ioctl/overwrite_all_of_destination (0.02s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
            --- FAIL: TestCopyRange/tmp/ioctl/overwrite_part_of_destination (0.02s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
            --- FAIL: TestCopyRange/tmp/ioctl/destination_gets_expanded_as_it_is_being_written_to (0.02s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
            --- FAIL: TestCopyRange/tmp/ioctl/unaligned_source_offset_unaligned_size (0.01s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
            --- FAIL: TestCopyRange/tmp/ioctl/unaligned_source_offset_aligned_size (0.01s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
            --- FAIL: TestCopyRange/tmp/ioctl/unaligned_destination_offset_unaligned_size (0.02s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
            --- FAIL: TestCopyRange/tmp/ioctl/unaligned_destination_offset_aligned_size (0.02s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
            --- FAIL: TestCopyRange/tmp/ioctl/aligned_offsets_unaligned_size (0.02s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
            --- FAIL: TestCopyRange/tmp/ioctl/last_block (0.01s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
            --- FAIL: TestCopyRange/tmp/ioctl/whole_file_copy_block_aligned (0.01s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
            --- FAIL: TestCopyRange/tmp/ioctl/whole_file_copy_not_block_aligned (0.01s)
                filesystem_copy_range_test.go:332: inappropriate ioctl for device
FAIL
FAIL	github.com/syncthing/syncthing/lib/fs	1.604s
FAIL
```

Looks like the ioctl that's used for the new copyRange functionality is not available on ppc64le / POWER.
The build succeeds on all other architectures supported by fedora (x86_64, i686, armv7hl / armhfp, aarch64, s390x / System Z).

--- a/lib/fs/basicfs_copy_range_ioctl.go
+++ b/lib/fs/basicfs_copy_range_ioctl.go
@@ -4,7 +4,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
-// +build linux,!ppc,!ppc64,!ppc64le 
+// +build linux,!ppc,!ppc64,!ppc64le
 
 package fs
 

--- a/lib/fs/basicfs_copy_range_ioctl.go
+++ b/lib/fs/basicfs_copy_range_ioctl.go
@@ -4,7 +4,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
-// +build linux
+// +build linux,!ppc64le 
 
 package fs
 

--- a/lib/fs/basicfs_copy_range_ioctl.go
+++ b/lib/fs/basicfs_copy_range_ioctl.go
@@ -4,7 +4,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
-// +build linux,!ppc64le 
+// +build linux,!ppc,!ppc64,!ppc64le 
 
 package fs
 

